app/CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# ods_flutter Demo Application
+
+All notable changes done in Demo application of ods_flutter library will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+## [Unreleased](https://github.com/Orange-OpenSource/ods-flutter/compare/0.1.1...main)
+
+- Update navigation in app using get
+  package ([#127](https://github.com/Orange-OpenSource/ods-flutter/issues/127))
+- Create components list
+  screen ([#132](https://github.com/Orange-OpenSource/ods-flutter/issues/132))
```

Comment:
It would be great to have only one line per change and I think we have to add sections (Added, Changed, ...) to respect the format. This is done like this in the Android changelog.